.travis.yml

```diff
@@ -54,10 +60,16 @@ jobs:
       before_install: *use-cmake
       env: *Debug
 
-before_script:
-  - export PATH=$(echo "$PATH" | sed -e 's/:\/usr\/lib\/ccache//')
```

Comment:
Where's this gone?

Reply:
To the annals of history!

That is to say, it doesn't seem to be needed. Caching still works fine.

Reply:
The `/usr/lib/ccache` directory contains a bunch of "fake" compilers that are wrapped in `ccache`. We use `CMAKE_<LANG>_COMPILER_LAUNCHER` instead of these.

Reply:
I mean, ccache seems to be working fine, so less code amirite?

Reply:
Still needs to go back.

Reply:
Could you remind me what it does though?

Reply:
It means cmake will detect the real compiler, not the `ccache` wrapped compiler. So, for example if we want to use a specific compiler version in CI then we aren't relying on there being a `ccache` wrapped version.